client/datasync-client/example/index.ts

```diff
@@ -31,7 +31,8 @@ $fh.sync.manage(datasetId, options, queryParams, metaData, function() {
       log.logger.error('Error when Saving Data', err);
     });
   }, function(err, data) {
-    log.logger.error('Error when Saving Data', err);
+    log.logger.error('Error when Saving Data', err,
```

Comment:
`log.logger` looks strange. Just map this to logger.

`const logger = loggerManager.getLogger();` or something like this.